crates/nu-protocol/src/signature.rs

```diff
@@ -46,6 +46,7 @@ pub enum Category {
     Conversions,
     Core,
     Custom(String),
+    Database,
```

Comment:
it appears `query db` also has this category but uses `Category::Custom` instead
https://github.com/nushell/nushell/blob/d43f4253e8f4591517f5ee32a9ccea4f69bb47d7/crates/nu-command/src/database/commands/query_db.rs#L27

maybe we should use `Custom` as well here or use this new `Database` in `query db`?

Reply:
The latter is exactly what I did :)

Reply:
i did not see it, my bad :eyes: